docs/content/en/docs/snippets/tasks/install.md

```diff
@@ -2,7 +2,33 @@
 
 ## Install version 0.6.0 and above
 
-In version 0.6.0 and later, you can install the Lifecycle Toolkit using the current release manifest:
+In version 0.6.0 and later, you can install the Lifecycle Toolkit using helm charts,
+or manifests.
+
+For installing the Lifecycle Toolkit via Helm chart:
+
+```shell
+helm repo add keptn-lifecycle-toolkit https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn-lifecycle-toolkit keptn-lifecycle-toolkit/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+To install a specific version, use the `--version <version>` falg as part of
+`helm upgrade --install` command.
+
+To list available versions:
+
+```shell
+helm repo update
+helm search repo keptn-lifecycle-toolkit
+```
+
+The `helm upgrade --install` command offers a flag called `--set`, which can be used to specify
+configuration options using the format key1=value1,key2=value2,....
+The full list of available flags can be found in the [helm-charts](https://github.com/keptn/lifecycle-toolkit/blob/main/helm/chart/README.md).
+
```

Comment:
Or you could download the chart value file and modify it using
`helm get values RELEASE_NAME [flags] > values.yaml`
https://helm.sh/docs/helm/helm_get_values/

and install adding --values=values.yaml